docs/faq.asciidoc

```diff
@@ -160,7 +160,7 @@ Debezium use the Kafka Connect framework. All of Debezium's connectors are Kafka
 
 When using Docker for Mac or Docker for Windows, the Docker containers run within a light-weight VM.
 In order to connect to Kafka from your host system, e.g. with a Kafka Consumer started in a test in your IDE,
-you need to specify your host system's IP address or host name as `ADVERTISED_HOST_NAME` for the Kafka container: `docker run -it --rm --name kafka -p 9092:9092 -e ADVERTISED_HOST_NAME=<%YOUR_HOST_NAME%> --link zookeeper:zookeeper debezium/kafka:{debezium-docker-label}`.
+you need to specify your docker VM's host system's IP address or host name as `ADVERTISED_HOST_NAME` for the Kafka container: `docker run -it --rm --name kafka -p 9092:9092 -e ADVERTISED_HOST_NAME=<%YOUR_DOCKER_VM_HOST_NAME%> --link zookeeper:zookeeper debezium/kafka:{debezium-docker-label}`.
```

Comment:
Interesting, how would that work? AFAIK, the Docker host isn't reachable from the host itself?

Reply:
@krishnaaravacadreon ping :)